lib/storage/metadata/mongoclient/MongoClientInterface.ts

```diff
@@ -2753,14 +3291,22 @@ class MongoClientInterface {
 
     getIndexingJobs(log, cb) {
         // list active createIndexes jobs
-        this.adminDb.command({
+        this.adminDb?.command({
```

Comment:
if adminDb is null-ish, `this.adminDb?.command` will return null as well, and we will still crash and not call the callback, right?
-> either assert type (`this.adminDb!.`) or yield an error early?